ci/axis/rapids-core-base-runtime.yaml

```diff
@@ -0,0 +1,33 @@
+BUILD_IMAGE:
+  - rapidsai/rapids-core
+  - rapidsai/rapids-core-nightly
```

Comment:
these should be `rapidsai/rapidsai-core` and `rapidsai/rapidsai-core-nightly`

and update the file name and exclusions